staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -264,19 +310,42 @@ func (q *delayingType) waitingLoop() {
 	}
 }
 
+// addImmediate handles how we should perform an immediate add - whether we allow preempting or not.
```

Comment:
Comment and function name don't match any more.

Can you say either in the comment or by changing the function's name what its expectations about the lock are?

Reply:
Ok have updated this to match.